jena-core/src/test/java/org/apache/jena/ttl_test/turtle/parser/JavaCharStream.java

```diff
@@ -729,7 +730,7 @@ public void Done()
    * @param newLine the new line number.
    * @param newCol the new column number.
    */
-
+  
```

Comment:
Not sure if GitHub UI, or if these whitespaces were added intentionally :+1:

Reply:
This is automatically generated by different versions of javacc.

(re-running the generator, same version, gives no changes which is helpful)